internal/terraform/terraform.go

```diff
@@ -0,0 +1,194 @@
+package terraform
+
+import (
+	"fmt"
+	"io/fs"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/coopnorge/mage/internal/devtool"
+)
+
+
+// IsTerraformProject returns true if a directory contains a go module.
```

Comment:
```suggestion
// IsTerraformProject returns true if a directory contains a Terraform project.
```

Please take a look at all the comments, there are a lot of copy-paste errors